src/subscription_manager/managercli.py

```diff
@@ -744,6 +744,10 @@ def _do_command(self):
 
         try:
             self._validate_options()
```

Comment:
Subsequently, I think we should distinguish between the following cases:
 - identity cert is completely absent (neither cert.pem nor key.pem exists on the filesystem)
 - identity cert was unable to be loaded

One way to accomplish this would be to add an `is_present()` method to the certificate, and then we can use `is_present()` to emit the message:

    This system is not yet registered. Try 'subscription-manager register --help' for more information.

If the certificate is present, we can then ask if it's valid, emitting:

     Consumer identity either does not exist or is corrupted. Try register --help